components/elm/src/biogeophys/SedYieldMod.F90

```diff
@@ -209,8 +209,8 @@ subroutine SoilErosion (bounds, num_soilc, filter_soilc, &
                      KE_DT = Ptot * fungrvl * max(8.95_r8+8.44_r8*log10(Ie), 0._r8)
                      ! leaf drip power
                      Dl = max(qflx_leafdrip(p)*dtime, 0._r8)      ! mm
-                     KE_LD = max(15.8_r8*sqrt(0.5_r8*(htop(p)+hbot(p)))-5.87_r8, 0._r8) * &
-                        fungrvl * Dl
+                     KE_LD = max(15.8_r8*sqrt(0.5_r8*min(htop(p)+hbot(p),40._r8))-5.87_r8, &
```

Comment:
This line has nested `max()` and `min()`. Let's split this long line and introduce a few temporary variables so the code is easy to read.